src/components/transactions/Repay/CollateralRepayModalContent.tsx

```diff
@@ -202,6 +204,13 @@ export function CollateralRepayModalContent({
             collateral to perform this action
           </Trans>
         );
+      case ErrorType.FLASH_LOAN_NOT_AVAILABLE:
+        return (
+          <Trans>
+            A flash loan is required to perform the repay, but this asset is not availalbe to use
```

Comment:
Due to health factor impact, a flashloan is required to perform this transaction, but Aave Governance has disabled flashloan availability for this asset. Try lowering the amount or supplying additional collateral.